jwst/assign_wcs/nirspec.py

```diff
@@ -256,29 +256,97 @@ def get_open_fixed_slits(input_model):
     return slits
 
 
-def get_open_msa_slits(msa_status):
+def get_open_msa_slits(msa_file, msa_metadata_id=88):
```

Comment:
I think `msa_metadata_id` should be a required argument, so don't give a default value.